iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/utils/IOUtils.java

```diff
@@ -117,6 +118,31 @@ public static String readString(
     return null;
   }
 
+  public static Pair<String, Boolean> readAuthString(
```

Comment:
add java doc for this method,  at least that like: for IOTDB 1.2, the username's length will be stored as a negative number.